BitmovinYospacePlayer/Sources/BitmovinYospacePlayer/Helper/Events/BitmovinEventsApiProxy.swift

```diff
@@ -0,0 +1,15 @@
+import BitmovinPlayerCore
+import Combine
+import Foundation
+
+class BitmovinEventsApiProxy: PlayerEventsApi {
+    let eventBus: EventBus
+
+    init(eventBuss: EventBus) {
```

Comment:
Is the 2nd `s` in `eventBuss` intentional or a typo? 😄

Reply:
https://github.com/bitmovin/bitmovin-player-ios-integrations-yospace/pull/130#discussion_r2627721694